db-commons/src/main/scala/org/bitcoins/db/AppConfig.scala

```diff
@@ -156,8 +161,10 @@ abstract class AppConfig extends StartStopAsync[Unit] with BitcoinSLogger {
 
   lazy val slickDbConfig: DatabaseConfig[JdbcProfile] = {
     Try {
-      DatabaseConfig.forConfig[JdbcProfile](path = s"bitcoin-s.$moduleName",
-                                            config = config)
+      val c = DatabaseConfig.forConfig[JdbcProfile](path =
+                                                      s"bitcoin-s.$moduleName",
+                                                    config = config)
+      c
```

Comment:
what's the point of this val

Reply:
I was printing stuff related to it and i guess i forgot to remove it.